test/Chainweb/Test/Pact/PactReplay.hs

```diff
@@ -112,7 +113,7 @@ testMemPoolAccess = mempty
         outtxs <-
           mkTestExecTransactions
             "sender00" "0" kp0
-            nonce 10000 0.00000000001
+            nonce 10_000 0.000_000_000_01
```

Comment:
I wonder if a hop was intended, not ten hops?

Reply:
who knows